### ISSUE TITLE
Add normalization option to confusion matrix

### ISSUE THREAD

**Issue by kmax12** [alteryx/evalml#432]
For unbalanced classes it can be helpful to color by normalized values rather than absolutes for each true class.

What if we added a parameter to the plot and generate confusion matrix data functions to support coloring by normalized values? We could also include both the normalized and absolute info in the tool tip the users see on hover regardless of this parameter. 

Sklearn offers this ability for reference here: https://scikit-learn.org/stable/auto_examples/model_selection/plot_confusion_matrix.html
